lib/publiccloud/utils.pm

```diff
@@ -269,4 +270,14 @@ sub prepare_ssh_tunnel {
     assert_script_run "touch $ssh_sut; chmod 777 $ssh_sut";
 }
 
+sub kill_packagekit {
+    my ($instance) = @_;
+    my $ret = $instance->ssh_script_run(cmd => "pkcon quit", timeout => 120);
+    if ($ret) {
+        # Older versions of systemd don't support "disable --now"
+        $instance->ssh_script_run(cmd => "systemctl stop packagekitd");
+        $instance->ssh_script_run(cmd => "systemctl disable packagekitd");
```

Comment:
packagekit is annoyingly resilient - Stopping and disabling `packagekitd` is not enough, if you really want to stop it, we also need to `mask` it

```suggestion
        $instance->ssh_script_run(cmd => "systemctl disable packagekitd");
        $instance->ssh_script_run(cmd => "systemctl mask packagekitd");
```